lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc

```diff
@@ -0,0 +1,208 @@
+// Copyright 2008 Google Inc.
```

Comment:
I liked the cartoon.. I did create a fork from Mixxx (
https://github.com/djsteph/mixxx-1) and created a branch (
https://github.com/djsteph/mixxx-1/tree/Mixtrack-3-Pro-v1.4---AutoLoop-mapping)
on my local repository. Where I think I messed up is synchronizing my fork
with the recent change from Mixxx. Just not sure how to do that..

In any event, here are the updated files attached in the zip file

Thanks
Steph

Stéphane Morin
Maison: 450 510-0622
Cellulaire: 514 243-5876

On Wed, Jul 6, 2016 at 12:17 PM, Be <notifications@github.com> wrote:

> In lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc
> <https://github.com/mixxxdj/mixxx/pull/976#discussion_r69762653>:
>
> > @@ -0,0 +1,208 @@
> > +// Copyright 2008 Google Inc.
>
> I think you've confused git branches and GitHub forks. They're not the
> same thing. Forking creates a clone of the git repository on GitHub's
> server; branching creates a new place to work within a repository.
> Generally, the workflow for contributing an update is to create a new
> branch in your local git repository (which was forked from mixxxdj/mixxx
> originally), merge the latest changes from mixxxdj/mixxx's master branch
> into your new branch, then commit your changes and open a pull request.
>
> But yeah, I can make a clean branch and submit a PR for you.
>
> http://xkcd.com/1597/ :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mixxxdj/mixxx/pull/976/files/aaf28730258d46331dbcff508caa9e1c4bb01e8a..aff0a5ebf86da603deb17b9163db74d92b7cb337#r69762653>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQDKyb4IVi4EhU2Pky3dvPUMP6scR_E-ks5qS9U3gaJpZM4JF-qh>
> .
>